orc8r/cloud/go/tools/swaggergen/main.go

```diff
@@ -100,6 +103,19 @@ func main() {
 		glog.Fatalf("Error parsing swagger spec dependency tree: %v\n", err)
 	}
 
+	if *generateStandAloneSpec && *targetFilepath != "swagger-common.yml" {
+		// Retrieve spec service name
+		splitPath := strings.Split(*targetFilepath, "/")
+		outFile := splitPath[len(splitPath)-4] + ".swagger.v1.yml"
+		outPath := filepath.Join(os.Getenv("MAGMA_ROOT"), "orc8r/cloud/swagger/specs/standalone", outFile)
+		err = generate.GenerateStandAloneSpec(*targetFilepath, specs, outPath)
+		if err != nil {
+			// Swallow GenerateStandAloneSpec error because build should
```

Comment:
In this case, we actually want to fatal. Since it's a tool, it's better to fail fast and loudly